nucypher/cli/options.py

```diff
@@ -48,6 +48,7 @@
 option_force = click.option('--force', help="Don't ask for confirmation", is_flag=True)
 option_gas_price = click.option('--gas-price', help="Set a static gas price (in GWEI)", type=GWEI)
 option_gas_strategy = click.option('--gas-strategy', help="Operate with a specified gas price strategy", type=click.STRING)  # TODO: GAS_STRATEGY_CHOICES
+option_key_material = click.option('--key-material', help="A pre-secured hex-encoded secret to use for private key derivations", type=click.STRING)
```

Comment:
Instead of directly inputting the key material here, an alternative is to pass a file path that contains the key material. The advantage here is not disclosing the key material from the shell history. Geth does something similar for passwords (see https://geth.ethereum.org/docs/interface/managing-your-accounts)

Reply:
Great suggestion.  I'll make the changes.